posts/inside-rust/leadership-council-repr-selection@5.md

```diff
@@ -0,0 +1,22 @@
++++
+layout = "post"
+date = 2025-03-25
+title = "Leadership Council March 2025 Representative Selections"
+author = "Eric Huss"
+team = "Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>"
++++
+
+The March 2025 selections for [Leadership Council] representatives have been finalized. The compiler team has chosen [Josh Stone] as their new representative. [Eric Huss] and [James Munns] will continue to represent [Devtools] and [Launching Pad] respectively.
+
+We'd like to give our thanks to the outgoing representative [Eric Holk] for his many substantial and valuable contributions.
```

Comment:
@eholk: Obviously your contributions have been enormous.  Anything you'd like for us to highlight here?

Reply:
Maybe something along the lines of "Eric helped demonstrated and helped established some of the core values of the Council, including transparency and accessibility of Council members. He drove the initial Project Director elections, and also strongly supported using project resources to advance the project, such as through project grants and developing support staff." I'm not sure, it probably needs some word smithing. Feel free to add your own thoughts!